test/t_octave.m

```diff
@@ -21,11 +22,12 @@
 endfunction
 
 
-opt.algorithm = NLOPT_LD_MMA
-%  opt.algorithm = NLOPT_LN_COBYLA
+%opt.algorithm = NLOPT_LD_MMA
+opt.algorithm = NLOPT_LN_SBPLX ;
 opt.lower_bounds = [-inf, 0]
+opt.upper_bounds = [10, inf]
 opt.min_objective = @myfunc
-opt.fc = { (@(x) myconstraint(x,2,0)), (@(x) myconstraint(x,-1,1)) }
-opt.fc_tol = [1e-8, 1e-8];
+%opt.fc = { (@(x) myconstraint(x,2,0)), (@(x) myconstraint(x,-1,1)) }
+%opt.fc_tol = [1e-8, 1e-8];
```

Comment:
Seems like these changes were included accidentally?

Reply:
You're right it happened accidentally.